Engine/source/gfx/bitmap/ddsFile.cpp

```diff
@@ -178,7 +178,7 @@ U32 DDSFile::getSizeInBytes( GFXFormat format, U32 height, U32 width, U32 mipLev
       "DDSFile::getSizeInBytes - Must be a Block Compression format!" );
 
    // From the directX docs:
-   // max(1, width � 4) x max(1, height � 4) x 8(DXT1) or 16(DXT2-5)
+   // max(1, width ÷ 4) x max(1, height ÷ 4) x 8(DXT1) or 16(DXT2-5)
```

Comment:
Your code editor isn't handling unicode properly and has mangled lines 181 and 126.